### PR TITLE
Updated introduction

### PR DESCRIPTION
**Pull request by Jceciliani**
Added introduction for clarity of purpose. 
Made minor text fixes.

N/A

### CODE REVIEW

**Comment by Jceciliani** [comment]
@swift-ci Please smoke test.

File: docs/ContinuousIntegration.md [hunk 1]
```diff
@@ -17,11 +17,11 @@
...
+In order for the Swift project to be able to advance quickly, it is important that we maintain a green build [[1]](#footnote-1). In order to help maintain this green build, the Swift project heavily uses pull request (PR) testing. Specifically, an important general rule is that **all** non-trivial checkins to any Swift Project repository should at least perform a [smoke test](#smoke-testing) if simulators will not be impacted *or* a full [validation test](#validation-testing) if simulators may be impacted. If in addition one is attempting to make a source breaking change across multiple repositories, one should follow the cross repo source breaking changes workflow. We now continue by describing the Swift system for Pull Request testing, @swift-ci:
```
Comment (gottesmm) [inline docs/ContinuousIntegration.md]: What did you do to this line?

Comment (Jceciliani) [reply docs/ContinuousIntegration.md]: Five lines down. 
From : any Swift Project repository should should at least perform a [smoke test]
To: any Swift Project repository should at least perform a [smoke test]

Also, added a comma after repositories 3 lines from the bottom.

**Comment by Jceciliani** [comment]
@swift-ci smoke test

**Comment by gottesmm** [comment]
@Jceciliani until you have commit access, swift-ci won't listen to you. But this LGTM.

**Comment by gottesmm** [comment]
@swift-ci smoke test and merge

**Pull request merged**